provider-tutorials/provider-cli.md

```diff
@@ -229,3 +231,138 @@ Run `ya-provider keystore --help` to see possible subcommands
+   It doesn't matter from which requestor node request is made.
+3. `partner`: allows outbound access for any manifests, as long as requestor node is a partner to trusted organisation.
+   Trust is gained when provider sets Partner rule for root certificate of trusted organisation.
+   Provider can have multiple `partner` rules for different [golem certificates](../requestor-tutorials/vm-runtime/golem-certificates.md).
+   To match this rule, requestor should get his [node descriptor signed](../requestor-tutorials/vm-runtime/golem-certificates.md) by partner to trusted organisation.
+   It doesn't matter which payload is being requested.
+
+Every rule can operate in three different modes:
+
+1. `all`: all domains listed in the manifest can be accessed for outbound traffic.
+2. `whitelist`: only whitelisted domains can be accessed for outbound traffic.
+3. `none`: no outbound traffic is allowed.
+
+> **Warning**
+> Remember, it is your responsibility as a provider to decide which entities you trust and the level of Outbound Network access you want to enable for them.
+
+By default, the following rules and modes are set during installation:
+
+- Everyone: Whitelist mode (every requestor can access outbound until domain is whitelisted).
+- AuditedPayload: All mode (Golem root certificate is trusted).
+- Partner: All mode (another Golem root certificate is trusted).
+
+#### Listing Rules
+
+To list currently set rules, you can use the `ya-provider rule list` command. By default, it displays the rules in a table format:
+
+```
+$ ya-provider rule list
+Outbound status: enabled
+┌──────────────────┬─────────────┬───────────────┬───────────────┐
+│  rule            │  mode       │  certificate  │  description  │
+├──────────────────┼─────────────┼───────────────┼───────────────┤
+│  Everyone        │  whitelist  │               │               │
+│  AuditedPayload  │  all        │  55e451bd     │               │
+│  Partner         │  all        │  80c84b27     │               │
+└──────────────────┴─────────────┴───────────────┴───────────────┘
+```
+
+You can also retrieve the rules in JSON format by adding the `--json` flag:
+
+```
+$ ya-provider rule list --json
+{
+  "outbound": {
+    "enabled": true,
+    "everyone": "whitelist",
+    "audited-payload": {
+      "55e451bd1a2f43570a25052b863af1d527fe6fd4bfd1482fdb241596432477f20eb2b2f3801fb5c6cd785f1a03c43ccf71fd8cdf0a974d1296be2326b0824673": {
+        "mode": "all",
+        "description": ""
+      }
+    },
+    "partner": {
+      "80c84b2701126669966f46c1159cae89c58fb088e8bf94b318358fa4ca33ee56d8948511a397e5aba6aa5b88fff36f2541a91b133cde0fb816e8592b695c04c3": {
+        "mode": "all",
+        "description": ""
+      }
+    }
+  }
+}
+```
+
+#### Enabling and Disabling Outbound Traffic
+
+You can enable or disable the entire outbound traffic regardless of other rules settings by using the following commands:
```

Comment:
can you indeed _enable_ "the entire network traffic" regardless of other rules? or just disable it?